infrastructure/sentry-subgraph/sepolia.subgraph.yaml

```diff
@@ -45,6 +45,12 @@ dataSources:
           handler: handleStakedV1
         - event: UnstakeV1(indexed address,uint256,uint256)
           handler: handleUnstakeV1
+        - event: NewPoolSubmission(indexed uint256,indexed address,uint256,uint256)
+          handler: handleNewPoolSubmission
+        - event: UpdatePoolSubmission(indexed uint256,indexed address,uint256,uint256,uint256,uint256)
+          handler: handleUpdatePoolSubmission
+        - event: PoolRewardsClaimed(indexed uint256,indexed address,uint256,uint256)
+          handler: handlePoolRewardsClaimed
       file: ./src/referee.ts
```

Comment:
Should this file also have these new events?

 event: AssertionSubmittedV2(indexed uint256,indexed uint256,bytes,bool)
          handler: handleAssertionSubmittedV2
        - event: RewardsClaimedV2(indexed uint256,indexed uint256,uint256)
          handler: handleRewardsClaimedV2
        - event: AssertionCancelled(indexed uint256,indexed uint256)
          handler: handleAssertionCancelled

Reply:
Yes good catch.
Its added in the follow up PR https://github.com/xai-foundation/sentry/pull/281